src/main/resources/cars.postman_collection.json

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
Please add some negative test cases. For example:
- Add a car with null brand or null country.
- Update a car with null brand or null country.
- Get a car with wrong  ID
- Delete a car that was already delete ( do not exist )